apps/src/StudioApp.js

```diff
@@ -246,6 +249,28 @@ StudioApp.prototype.configure = function (options) {
 };
 
 /**
+ * Creates a redux store for this app, while caching the set of app specific
+ * reducers, so that we can recreate the store from scratch at any point if need
+ * be
+ * @param {object} reducers - App specific reducers
```

Comment:
It might be worth calling out here that it's valid to pass null.  Do apps that don't have any reducers to pass have to call configureRedux?